awx_collection/tests/integration/targets/tower_workflow_job_template/tasks/main.yml

```diff
@@ -145,6 +146,36 @@
     unified_job_template: "{{ jt1_name }}"
     workflow: "{{ wfjt_name }}"
 
+- name: Fail if no name is set for approval
+  tower_workflow_job_template_node:
+    identifier: approval_test
+    approval_node:
+      description: "{{ approval_node_name }}"
+    workflow: "{{ wfjt_name }}"
+  register: no_name_results
+  ignore_errors: true
+
+- assert:
+    that:
+      - "no_name_results.msg == 'Approval node name is required to create approval node.'"
+
+- name: Create approval node
+  tower_workflow_job_template_node:
+    identifier: approval_test
+    approval_node:
+      name: "{{ approval_node_name }}"
+      timeout: 900
+    workflow: "{{ wfjt_name }}"
```

Comment:
Can we also test a state == absent in here to make sure that is working?

Reply:
Added absent the node.